packages/devcmd-cli/src/index.ts

```diff
@@ -73,6 +73,19 @@ async function startProcess(command: string, args: Array<string>, dirPath: strin
 async function runInDevCmdsDir(dirPath: string): Promise<void> {
   const [, , ...args] = process.argv;
 
+  if (args.length > 0) {
+    const cmdName = args[0].trim();
+
+    if (cmdName.length > 2 && cmdName.indexOf("--") === 0) {
+      try {
+        await startProcess("node", ["-e", `require('devcmd/from-cli').runReserved('${cmdName.slice(2)}')`], dirPath);
```

Comment:
with this code, you can't pass any further arguments to the reserved command. This isn't an issue for the two reserved commands we have right now, but I'd prefer to minimize unnecessary changes to devcmd-cli down the line.

please include the rest of `args` and also pass them to `runReserved` -- probably best also do this as separate arguments like we do for regular commands below, otherwise you need to figure out how to quote-escape them correctly :D

Reply:
Ahhh right, I hadn't realized that issue would of course affect this call as well. 

for any other readers: just like calling `devcmd --help` is a problem because it would trigger `node --version` before these changes, even with this `runReserved` call we could run into the same issue if the user tries `node --list --help`.

@lieberlois, I hadn't thought of this before, but we can probably side-step this entire issue by passing the [double-dash option (`--`)](https://nodejs.org/api/cli.html#cli_1) in the args list to node at the right place.
The double-dash option is a common feature of many *nix CLI programs to signal the "end of options", i.e. all arguments after the double dash argument are not interpreted as options to the command itself.

So in our case, we could do something like the following:

```ts
await startProcess(
  "node", 
  [
    "-e", 
    `require('devcmd/from-cli').run(...process.argv.slice(1))`, 
    "--",      // this is new
    ...args], 
  dirPath);
```

Reply:
Exactly what I had in mind aswell 👍 

This now works properly:

```sh
node ➜ /workspaces/devcmd/dev (fix/30-help-list) $ yarn devcmd --help
yarn run v1.22.5
$ /workspaces/devcmd/dev/node_modules/.bin/devcmd --help
devcmd v0.0.8Usage: devcmd <task>
Use devcmd --list to show available tasks.
Done in 0.21s.
node ➜ /workspaces/devcmd/dev (fix/30-help-list) $ yarn devcmd example_cmd
yarn run v1.22.5
$ /workspaces/devcmd/dev/node_modules/.bin/devcmd example_cmd
devcmd v0.0.8: cmd "example_cmd"
Example command for single-package-json example
Starting process: "node" ["-v"]
v12.22.4
Process "node" exited successfully.
Beginning parallel execution of 2 processes...
<nodeVersion> Starting process: "node" ["-v"]
<npmVersion> Starting process: "npm" ["--version"]
<nodeVersion> v12.22.4
<nodeVersion> Process "node" exited successfully.
<npmVersion> 6.14.14
<npmVersion> Process "npm" exited successfully.
Finished parallel execution.
Done in 5.10s.
```